Prox/Prox/RemoteConfigKeys.swift

```diff
@@ -28,36 +28,40 @@ class RemoteConfigKeys {
     // Any previously fetched and cached config would be considered expired because it would have been fetched
     // more than remoteConfigCacheExpiration seconds ago. Thus the next fetch would go to the server unless
     // throttling is in progress. The default expiration duration is 43200 (12 hours).
-    public static let remoteConfigCacheExpiration = "remote_config_cache_expiration"
+    public static let remoteConfigCacheExpiration = RemoteConfigDouble(key: "remote_config_cache_expiration", defaultValue: 12.0 * 60 * 60)
```

Comment:
nit: feel like we should use some constants like `TIME_IN_HOUR` here or something: multiplying by values of time always feels so fragile!